src/app/+search-page/search.service.ts

```diff
@@ -101,6 +110,11 @@ export class SearchService {
         });
     });
 
+    itemsRD.hasSucceeded.subscribe((success: boolean) => {
```

Comment:
If you need requestPending and responsePending to actually work for this mock search service, I'd simply use itemsRD.isRequestPending and isResponsePending in the new RemoteData object instead of this setup with the BehaviorSubjects